Purchasing.Mvc/Services/NotificationService.cs

```diff
@@ -399,6 +403,10 @@ public void OrderAddAttachment(Order order, User actor)
             var users = order.OrderTrackings.Select(a => a.User).Distinct().ToList();
             foreach (var ot in users)
             {
+                if (!ot.IsActive) //Not active, we don't want to email them
```

Comment:
@jSylvestre sure, or you could just add a where to the query itself, like `var activeUsers = order.OrderTrackings.Where(a=>a.User.IsActive).Select(a => a.User).Distinct().ToList();`  Don't know which would be more efficient, I'd assume the db query was at least no worse